granian/_imports.py

```diff
@@ -5,5 +5,7 @@
 
 try:
     import watchfiles
+    from watchfiles import BaseFilter
```

Comment:
maybe we can use `watchfiles.BaseFilter` in *server.py* and leave this untouched? wdyt?

Reply:
Ah, I wasn't aware we could type hint to a potential `None.BaseFilter` but it seems to work.